inferentia/scripts/gen_triton_model.py

```diff
@@ -336,7 +342,8 @@ def _validate_output_dict(self, expected_count):
     return init_impl
 
 
-def get_tensorflow_execute_impl():
+def get_tensorflow_execute_impl(enable_batching,
```

Comment:
Same here.. I don't think you need to pass enable_batching flag here. Just look whether execute function was called with single request or multiple.

Reply:
I was trying to speed up execution for the default case (no batching) so there doesn't need to be extra checks

Reply:
You can speed up the execution when the server sent only a single request to the backend. That is even if the dynamic batching is enabled, the server does not always send multiple requests to the backend. For instance, if the delay time expired and only a single request was received by the server, we don't need to perform extra logic...